android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -64,39 +64,41 @@
     // Update the KMW banner height
     function setBannerHeight(h) {
       var kmw=com.keyman.singleton;
+      var osk = kmw.osk;
       if (h > 0) {
-        var osk = kmw.osk;
-        osk.banner.height = Math.ceil(h / window.devicePixelRatio);
+        // The banner itself may not be loaded yet.  This will preemptively help set
+        // its eventual display height.
+        com.keyman.osk.Banner.DEFAULT_HEIGHT = h;
```

Comment:
@jahorton is this safe?

Reply:
I got this from iOS:

https://github.com/keymanapp/keyman/blob/8843828879fedb3a9f6302aa115a8b08e7994982/ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html#L79-L87

Reply:
I don't think iOS has to worry about the `h / window.devicePixelRatio` part... are we sure that the pixel-ratio bit doesn't apply here?